tests/ScheduleElementTest.php

```diff
@@ -20,14 +20,13 @@ protected function setUp(): void
     protected function assembleElement(array $options): ScheduleElement
     {
         $element = new ScheduleElement('test', $options);
+        $element->render(); // Forces also assembly of any content
```

Comment:
Why `render()` instead of `ensureAssembled()`?

Reply:
Because `render()` is calling `ensureAssembled()` automatically before rendering other elements that provide `renderWrapped()` & `renderUnwrapped()` implementation.

Reply:
Not sure why you need to change this. It doesn't make a difference (anymore?) if I run the tests.